packages/zoltan/src/zz/zz_struct.c

```diff
@@ -135,19 +135,6 @@ int proc;
 
   zz->ZTime = Zoltan_Timer_Create(ZOLTAN_TIMER_DEF);
 
-  /* Test that size_t is uniform on all processors */
```

Comment:
I'm pretty sure that other Trilinos code sends and receives arrays of `size_t` without checking whether `size_t` has uniform size on different processes, so yes, I agree that this is an ill-tested case we need not support.